### PR TITLE
Support IGNORE argument in TimeSeries commands

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

File: src/test/java/redis/clients/jedis/modules/timeseries/TimeSeriesTest.java [hunk 1]
```diff
@@ -288,6 +288,25 @@ public void testAdd() {
     }
   }
 
+  @Test
+  public void testCreateIgnore() {
+    assertEquals("OK", client.tsCreate("series-ignore",
+        TSCreateParams.createParams().ignore(5, 3)));
+    //System.out.println(client.tsInfo("series-ignore").getProperties()); // doesn't provide IGNORE info
```
Comment (tishun) [inline src/test/java/redis/clients/jedis/modules/timeseries/TimeSeriesTest.java:295]: NIT: Perhaps not need to submit this?

File: src/main/java/redis/clients/jedis/timeseries/RedisTimeSeriesCommands.java [hunk 2]
```diff
@@ -59,7 +73,14 @@ public interface RedisTimeSeriesCommands {
   long tsAdd(String key, long timestamp, double value);
 
   /**
-   * {@code TS.ADD key timestamp value [RETENTION retentionTime] [ENCODING [COMPRESSED|UNCOMPRESSED]] [CHUNK_SIZE size] [ON_DUPLICATE policy] [LABELS label value..]}
+   * {@code TS.ADD key timestamp value
```
Comment (uglide) [inline src/main/java/redis/clients/jedis/timeseries/RedisTimeSeriesCommands.java:76]: TS.INCRBY and TS.DECRBY should also be updated

**Review by uglide — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/timeseries/RedisTimeSeriesCommands.java [hunk 3]
```diff
@@ -59,7 +73,14 @@ public interface RedisTimeSeriesCommands {
   long tsAdd(String key, long timestamp, double value);
```
[the remark this answers is not in the document]
Comment (sazzad16) [reply src/main/java/redis/clients/jedis/timeseries/RedisTimeSeriesCommands.java:73]: Yes

File: src/main/java/redis/clients/jedis/timeseries/TSAlterParams.java [hunk 4]
```diff
@@ -17,6 +17,11 @@ public class TSAlterParams implements IParams {
   private Long retentionPeriod;
   private Long chunkSize;
   private DuplicatePolicy duplicatePolicy;
+
+  private boolean ignore;
```
[the remark this answers is not in the document]
Comment (sazzad16) [reply src/main/java/redis/clients/jedis/timeseries/TSAlterParams.java:21]: See comment below.

File: src/main/java/redis/clients/jedis/timeseries/TSAlterParams.java [hunk 5]
```diff
@@ -41,6 +46,13 @@ public TSAlterParams duplicatePolicy(DuplicatePolicy duplicatePolicy) {
     return this;
   }
 
+  public TSAlterParams ignore(long maxTimediff, double maxValDiff) {
+    this.ignore = true;
+    this.ignoreMaxTimediff = maxTimediff;
```
[the remark this answers is not in the document]
Comment (tishun) [reply src/main/java/redis/clients/jedis/timeseries/TSAlterParams.java:51]: Fair. Thanks

**Comment by sazzad16** [comment]
Resolved by #3860

**Pull request closed without merging**